client/my-sites/domains/domain-management/security/index.jsx

```diff
@@ -0,0 +1,191 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import Header from 'my-sites/domains/domain-management/components/header';
+import { domainManagementEdit } from 'my-sites/domains/paths';
+import { CompactCard } from '@automattic/components';
+import MaterialIcon from 'components/material-icon';
+import { getSelectedDomain } from 'lib/domains';
+import RenewButton from 'my-sites/domains/domain-management/edit/card/renew-button';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+import { getProductBySlug } from 'state/products-list/selectors';
+import {
+	getByPurchaseId,
+	isFetchingSitePurchases,
+	hasLoadedSitePurchasesFromServer,
+} from 'state/purchases/selectors';
+
+import './style.scss';
+
+const SSL_DISABLED = 'disabled';
+const SSL_PROVISIONING = 'provisioning';
+const SSL_ACTIVE = 'active';
+
+class Security extends React.Component {
+	header() {
+		return (
+			<Header onClick={ this.back } selectedDomainName={ this.props.selectedDomainName }>
+				{ this.props.translate( 'Domain security' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Review your domain security' )` *ES Score: 7*